src/components/Header/Header.tsx

```diff
@@ -1,3 +1,120 @@
-import React from "react";
+import AccessTimeOutlinedIcon from "@mui/icons-material/AccessTimeOutlined";
+import {
+  AppBar,
+  Input,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import { Box, Container } from "@mui/system";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
 
-export const Header = () => <div>Header</div>;
+import eng from "../../assets/icons/eng.png";
+import logo from "../../assets/icons/logo.png";
+import ukraine from "../../assets/icons/ukraine.png";
```

Comment:
Instead of using flags as image you can use emoji, for example 🇦🇩

Reply:
done